import-scripts/backup-eks-dbs.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+DBS=("cgds_gdac" "cgds_triage" "keycloak")
+LOCAL_BACKUP_DIR=/data/mysql-dumps
```

Comment:
I wonder if we want to add this directory to our setup system scripts (for configuring a "pipelines" node deployemnt)